src/pages/iou/request/step/IOURequestStepScan/index.js

```diff
@@ -177,6 +177,10 @@ function IOURequestStepScan({
     };
 
     const getScreenshot = useCallback(() => {
+        if (!cameraRef.current || !cameraRef.current.getScreenshot) {
```

Comment:
```suggestion
        if (!cameraRef.current) {
```

Is there a case where `cameraRef.current` would be set and `cameraRef.current.getScreenshot` not?